packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -349,6 +359,8 @@ Media.getMedia = (audio, video, config) => {
         resolve(stream);
       })
       .catch((err) => {
+        LoggerProxy.logger.error(`media/index->getMedia#navigator.mediaDevices.getUserMedia failed - ${err} (${err.constraint})`);
```

Comment:
This will output
![image](https://user-images.githubusercontent.com/3140325/83809829-f926c300-a66b-11ea-8549-a2dea4db52ec.png)